lib/logstash/outputs/elasticsearch.rb

```diff
@@ -362,11 +365,43 @@ def config_init(params)
   # Receive an array of events and immediately attempt to index them (no buffering)
   def multi_receive(events)
     wait_for_successful_connection if @after_successful_connection_done
-    retrying_submit map_events(events)
+    events_mapped = safe_interpolation_map_events(events)
```

Comment:
How do you think if we reject index name unresolved events and only bring back successful events?
In my [unmerged PR](https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1080/files#diff-221ddb62bde6d62966fab1051387ded7513fffd575cf0706a5f3a94ad2d31307R198), we had a great discussion with Ry and agreed to work with successful/submittable events here, send to DLQ rejected events wen filtering.